deploy/clusterrole.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
I guess it's not possible to encode this as a "Role" rather than a "ClusterRole", is it?

Reply:
no because what we want is, we want to give this to users in different namespaces.

Note that we don't create the ClusterRole within the operator. It is created prior to operator installation and in operator we just create ClusterRoleBindings for users.

Reply:
Ah right, ok. I thought it was for the MDC ServiceAccount :+1: